lib/middleware.js

```diff
@@ -58,7 +58,7 @@ export default function handleRequest(req, resp) {
       
         console.log('promises.to/ request', req.params);
         resp.render('promise', {
-          ...req.params,
+          ...p,
```

Comment:
because i was feeding the template from the request params instead of the parsed promise, the full promise text wasn't displaying when there were slashes in the date (because of the way express parses the params, which we are moving away from)